specifications/api/swagger.yaml

```diff
@@ -159,24 +159,6 @@ x-walletMnemonicSentence: &walletMnemonicSentence
     format: bip-0039-mnemonic-word{english}
   example: ["squirrel", "material", "silly", "twice", "direct", "slush", "pistol", "razor", "become", "junk", "kingdom", "flee", "squirrel", "silly", "twice"]
 
-x-byronWalletMnemonicSentence: &byronWalletMnemonicSentence
-  description: |
-    A list of mnemonic words.
-
-    Important:
-      - a 12-word mnemonic sentence will be treated as being a Daedalus Byron wallet.
-      - a 15-word mnemonic sentence will be treated as being a Yoroi Byron wallet.
-      - a 24-word mnemonic sentence will be treated as being a Hardware wallet (Trezor or Ledger).
```

Comment:
It does _not_ (from my PoV at least. We can see the wallet's style as a sub-resource of the "byron-wallet" resource. All endpoints that are common to byron-wallets can be reached directly with no sub-typing necessary. If there were endpoints only available to some of the byron wallets, then they could have been referenced under their corresponding sub-type). 

Note that this works okay only because they have a same resource representation in the end `ApiByronWallet`.